axis.go

```diff
@@ -46,7 +46,13 @@ type Axis struct {
 		draw.TextStyle
 
 		// Position is where the axis label string should be drawn.
-		Position draw.Position
+		// The default value is draw.PosCenter, displaying the label
+		// at the center of the axis.
+		// Use draw.PosRight to display the label string at the far
+		// right of an X-axis.
+		// Use draw.PosTop to display the label string at the top
+		// of an Y-axis.
```

Comment:
I don't think these are necessary.

Reply:
adapted to mention the `[-1,+1]` valid interval.

PTAL.